tests/unit_test/llm_test.py

```
@@ -2611,4 +2631,109 @@ def test_load_rerank_embedding_model_hf_cache(self):
+                {"content": "AI is cool", "is_failure": False},
+                0.5
+            ))
+            mock_processor_inst.logs = []
+            mock_query.return_value.first.return_value = mock_emb_metadata
+
+            await action_prompt_instance.execute(dispatcher, tracker, domain, **kwargs)
+
+            assert mock_processor_inst.llm_type == "openrouter"
+            mock_secret.assert_called_with("openrouter", "test_bot")
+
+            assert mock_processor_inst.llm_secret == "sk-router-key"
```

Ruff (0.15.9)

[error] Possible hardcoded password assigned to: "llm_secret"

(S105)